### ISSUE TITLE
Allegro Kurier Pocztex błąd pobrania

### ISSUE THREAD

**Issue by ZAK28ZAK** [allegro/allegro-api#9753]
Dzień dobry,
Przy próbie wygenerowania biletu pobraniowego dla metody Allegro Kurier Pocztex otrzymujemy błąd:
{
      "code": "COD_NOT_SUPPORTED",
      "details": "COD cannot be passed in the request.",
      "message": "Payment on delivery is not supported for the selected delivery option.",
      "path": "cod",
      "userMessage": "Płatność pobraniowa nie jest wspierana dla wybranej metody dostawy."
    }

co może być tego przyczyną?

W programie który używamy widzę "traceId": "9d7cdad2e57ec4f4" 

Czy to wystarczy Państwu do zweryfikowania problemu?

**Comment by MaciejFrackowiak** [comment]
Wskazana w requeście metoda dostawy to Allegro Kurier Pocztex (id=093d950b-8497-4cc7-aa2a-6081e5e7e7e2) - metoda ta nie wspiera opcji pobrania.

**Comment by ZAK28ZAK** [comment]
Ale jak to nie wspiera, przecież jest taka opcja w Allegro?
![image](https://github.com/user-attachments/assets/1d9e014e-7311-4bc0-a1a4-ec0a20dfeaa6)

**Comment by MaciejFrackowiak** [comment]
Oczywiście jest, tylko dla niej identyfikator jest zupełnie inny aniżeli wskazany id=093d950b-8497-4cc7-aa2a-6081e5e7e7e2.

**Comment by ZAK28ZAK** [comment]
To jaki identyfikator powinien być poprawny dla Allegro Kurier Pocztex Pobranie?

**Comment by MaciejFrackowiak** [comment]
id=193d950b-0082-408f-9192-6081e5e7e7e2

**Comment by ZAK28ZAK** [comment]
Od twórców naszego oprogramowania dostałem informację, że aby uruchomić Allegro Kurier Pocztex Pobranie trzeba mimo wszystko posiadać umowę własną z Pocztą Polską, tak faktycznie jest to wymyślone? Mogą Państwo potwierdzić?

**Comment by MaciejFrackowiak** [comment]
Tak, jest to prawda. Szczegóły opisane są na stronie [Pomocy Allegro](https://help.allegro.com/sell/pl/a/metody-dostawy-poczty-polskiej-i-pocztex-informacje-dla-sprzedajacych-lL55gykEqtk).
